cloudsync_onedrive.py

```diff
@@ -781,7 +781,7 @@ def _convert_to_event(self, change, new_cursor) -> Optional[Event]:
 
         ts = arrow.get(change.get('lastModifiedDateTime')).float_timestamp
         oid = change.get('id')
-        exists = not change.get('deleted')
+        exists = change.get('deleted') is None
```

Comment:
an empty dict is falsy in python.  why was this failing?

Reply:
sure, will add a comment to explain.